app/views/defence_requests/new.html.erb

```diff
@@ -1,2 +1 @@
-<%= render 'shared/form_template', form_type: 'new', object_name: 'defence_request' %>
-
+<%= render 'shared/form_template', form_type: 'new', object_name: 'defence_request' %>
```

Comment:
We refactored the form template so you will need to pass in `klass: DefenceRequest` after rebasing.